docker-compose-apisix.yml

```
@@ -25,3 +23,27 @@ services:
       - ./config/apisix/apisix.yml:/usr/local/apisix/conf/config.yaml
     depends_on:
       - etcd
+
+  keycloak:
+    image: quay.io/keycloak/keycloak:latest
+    depends_on:
+      - db
+    ports:
+      - 7080:7080
+      - 7443:7443
+    environment:
+      - KEYCLOAK_ADMIN=${KEYCLOAK_SVC_ADMIN:-admin}
+      - KEYCLOAK_ADMIN_PASSWORD=${KEYCLOAK_SVC_ADMIN_PASSWORD:-admin}
```

Suggest add a profile for this, I have made some more progress locally running keycloak independently so it would be helpful to not have to run this:
```suggestion
      - KEYCLOAK_ADMIN_PASSWORD=${KEYCLOAK_SVC_ADMIN_PASSWORD:-admin}
    profiles:
      - keycloak
```